tests/behaviour/steps/concept/type_/owns.rs

```diff
@@ -40,34 +40,6 @@ pub async fn set_owns(
             &tx.type_manager,
             &tx.thing_manager,
             attr_type,
-            Ordering::Unordered,
-        );
-        may_error.check_concept_write_without_read_errors(&res);
-    });
-}
-
-#[apply(generic_step)]
-#[step(expr = "{root_label}\\({type_label}\\) set owns: {type_label}[]{may_error}")]
```

Comment:
Is removing this intentional?

Reply:
Yes, concept api tests should be as much one-to-one command-to-apicall as possible. Using this shortcut is not correct anymore.